integrations/color-mode.yml

```diff
@@ -1,11 +1,13 @@
 name: color-mode
 description: Dark and Light mode for NuxtJS with auto detection
-long_description: ''
+long_description: '🌑 Dark and 🌕 Light mode with auto detection made easy with NuxtJS'
 repo: nuxt-community/color-mode-module
 npm: '@nuxtjs/color-mode'
 type: module
 github: 'https://github.com/nuxt-community/color-mode-module'
 website: 'https://github.com/nuxt-community/color-mode-module'
+demo: https://color-mode.nuxtjs.app/
+gif: https://user-images.githubusercontent.com/904724/79349768-f09cf080-7f36-11ea-93bb-20fae8c94811.gif
```

Comment:
We can use mp4 video format instead of gif to reduce size and also keep assets in top level assets (or repo) with cdnja

Reply:
yes agree but this module only has a gif and not an mp4 so we can either choose to include it or create another one. I will see if I can find one from a blog post I created